src/routes/resource-edit.js

```diff
@@ -60,19 +60,21 @@ class ResourceEditRoute extends Component {
       customEmbargoValue,
       customEmbargoUnit,
       customUrl,
-      isHidden
+      isVisible
     } = values;
 
     if (values.isSelected === false && model.package.isCustom) {
       destroyResource(model);
     } else if (values.isSelected === false) {
-      model.isSelected = !model.isSelected;
+      model.isSelected = false;
       model.customCoverages = [];
-      model.visibilityData.isHidden = !isHidden;
+      model.visibilityData.isHidden = false;
       model.identifiersList = [];
       model.identifiers = {};
```

Comment:
this comment is not related to this PR but not sure if we should have both identifiersList and identifiers in here.

Reply:
yes, that would be my expectation too considering that mod-kb accepts 'identifiers' which is a []

Reply:
considering there may be other fields and cases related to re-setting the resource, I am not going to make changes in this PR